Cargo.toml

```diff
@@ -13,7 +13,7 @@ features = ["std", "suggestions", "derive"]
 [dependencies]
 nix = "0.19.1"
 procfs = "0.9.1"
-caps = "0.5.1"
+caps = { git = "https://github.com/lucab/caps-rs", features = ["serde_support"] }
```

Comment:
Can you add a `rev` or `tag` and comment that you are waiting for caps to be released?